about.html

```diff
@@ -0,0 +1,97 @@
+  <header>
+    <nav>
+      <a href="index.html"><img class="logo" src="/images/logo.svg" alt="logo"></a>
+
+      <ul class="enlaces-menu">
+        <li><a href="about.html">About</a></li>
+        <li><a href="#program">Programs</a></li>
+        <li><a href="#">Join</a></li>
+        <li><a href="#">Sponsor</a></li>
+        <li><a href="index.html">News</a></li>
+        <li><a href="index.html" class="alert">Hempaing</a></li>
+      </ul>
+
+      <button class="ham" type="button">
+        <span class="br-1"></span>
+        <span class="br-2"></span>
+        <span class="br-3"></span>
+      </button>
+    </nav>
+  </header>
+  <section class="container-fluid --bs-gray-400  homeAbout">
+    <h1 class="home-page2 text-center">AGRO-INDUSTRY<br> LATAM SUMMIT 2021</h1>
+    <h4 class="card-text text-center cardAbout" id="cardAbout">
+      A place where world's industry leaders shows their newest product and innovations.
+      With forums about the legal status of the plant,
+      Expositions about processing machines,
+      and world recognized experts.</h4>
+    <strong class="text-center mb-5">Please, contact us per email for any further question regarding the event</strong>
+  </section>
+  <section class="container-fluid text-center bg-white p-5 logoSection">
+    <h2 class="h1">Agro-Indistry LATAM Summit 2021 logo</h2>
+    <span class="tumbnail"></span>
+    <p class="text-center p-5 m-5">The Hemp Ind. Global commit 2021 was decided through the ogo competition,
+      from 8th june to 7th July. After online public voting and a selection
+      commetee's judging process. The logo made by Sergio Valadez won the price.
+      After consult it with Mr. VALERA. with mister Valadez we optimizad price winner logo.
+      for this year submmit. After consouling with Mr valadez, we optimized the price winning logo.
+    </p>
+    <a><img src="/images/logo.svg" width="400" alt="winnerlogo" id="winnerlogo"></a>
+
+  </section>
+  <section class="container-fluid text-center mt-5 mb-5">
+    <h2 class="h1">See past Hemp Industry submmits</h2>
+    <p class="h2">Take a look at the las two Hemp Industries submmits in Tokio and Dubai</p>
+    <div class="row">
+      <div class="col-md-6">
+        <p class="tokio h1">Tokio 2018</p>
+      </div>
+      <div class="col-md-6">
+        <p class="dubai h1">Dubai 2019</p>
+      </div>
+    </div>
+  </section>
+
+  <footer class="cointainer-fluid m-0 bg-black text-white align-content-center  row">
+    <img src="/images/logo.svg" width="5" alt="logo evento" class="col center">
+    <div class="col text-center m-5">
+      <p>Hemp Industries de México.<br> todos los derechos reservados.<br>
+        Cumbres de Maltrata 155 Int. 2 <br> Col. Narvarte. CDMX CP 03020</p>
+
+      Contacto:<br>
+      sergioval@gmail.com<br>
+      55-5555-5555
+    </div>
+  </footer>
+  <script src="js/speakers.js"></script>
+  <script src="js/program.js"></script>
+  <script src="js/nav.js"></script>
+</body>
+
```

Comment:
- [ ] CONGRATULATIONS 👏  👏  on improving your styling 👍  however, the changes you made are only working locally but your deployed version still has some images path errors. I cloned your project and it was working but the online version no.
- ![image](https://user-images.githubusercontent.com/65068771/148298426-cd0b0524-fb9c-4b51-b5df-6d94a6be329a.png)
- ![image](https://user-images.githubusercontent.com/65068771/148298502-e52755c2-f8d8-46ea-a64e-2a432174da39.png)
- ![image](https://user-images.githubusercontent.com/65068771/148299373-b3c90a8a-c551-485b-b530-f55d1ffc1b04.png)

Reply:
Got it. I was missing some dots on the path.